packages/playwright-test/src/runner.ts

```diff
@@ -289,7 +288,7 @@ export class Runner {
 
   private _runPhaseFromOptions(options: RunOptions): RunPhase {
     const testFileMatcher = fileMatcherFrom(options.testFileFilters);
-    const testTitleMatcher = () => true;
+    const testTitleMatcher = options.testTitleMatcher;
```

Comment:
Do we take `options.testTitleMatcher` into account when running groups? I think a similar TODO as we have to `testFileFilters`?

Reply:
Currently we don't,  added TODO.